var/spack/repos/builtin/packages/cpp-logger/package.py

```diff
@@ -19,3 +19,4 @@ class CppLogger(CMakePackage):
     version("master", branch="master")
     version("0.0.1", tag="v0.0.1", commit="d48b38ab14477bb7c53f8189b8b4be2ea214c28a")
     version("0.0.2", tag="v0.0.2", commit="329a48401033d2d2a1f1196141763cab029220ae")
+    version("0.0.3", tag="v0.0.3", commit="398e6fa1eb4442cba94d46ecacfa47a426474387")
```

Comment:
Please reverse the ordering of these version directives.  The convention is shown https://spack.readthedocs.io/en/latest/packaging_guide.html#versions-and-fetching.

Reply:
Done